docker/development/Dockerfile

```diff
@@ -11,8 +11,9 @@ RUN for i in 1 2 3; do \
             iputils-ping \
             lcov \
             ninja-build \
-            python3-pip \
             python3.10 \
+            python3-pip \
+            python3.10-venv \
```

Comment:
Setting up venv inside docker seems really pointless to me: Docker *is* the tool manage diffent enviorentments per tool why stack another tool that serves essentially the same purpose but is less capable on top of it?